R/download_data_cube.r

```diff
@@ -127,4 +131,8 @@ download_abs_data_cube <- function(cat_no,
 
   writeBin(httr::content(download_object, "raw"), filepath)
 
+  message("File downloaded in ", filepath)
+
+  return(invisible(TRUE))
```

Comment:
Seems a waste of a return value. Perhaps instead of returning `TRUE` the path should be returned. That way, it can be supplied at the start of a pipe

Reply:
Good idea @HughParsonage. The `invisible(TRUE)` was my idea, this is better

Reply:
Sorry - I am not sure what this code would look like?

Reply:
Does it actually need an explicit return value?

Reply:
Hi @daviddiviny, I think this would just look like `return(invisible(filepath))`. Not a huge issue either way, but as @HughParsonage says if the path is returned then this function can be used at the start of a `%>%` chain

Reply:
The current pull request has no return value. Does it need one?
If it does, I don't know how to pipe to a conditional message.

Reply:
Hi @daviddiviny, at the moment it's invisibly returning `TRUE`. 

``` r
print(readabs::download_abs_data_cube(cat_no = "6291.0.55.003", cube = "EQ09"))
#> File downloaded in /var/folders/fy/y9xb4vvj7xvd8mr_ddb995800000gq/T//RtmptFEjEg/eq09.zip
#> [1] TRUE
```

<sup>Created on 2020-08-04 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

I find returning something (even if it's invisible) useful for testing, etc. 

If you change `return(invisible(TRUE))` to `return(invisible(filepath))` then you'd be able to do something like: 

```
readabs::download_abs_data_cube %>%
  unzip() %>%
  readxl::read_excel()
```

Reply:
Done.